orcid-core/src/main/java/org/orcid/core/manager/impl/WorkManagerImpl.java

```diff
@@ -240,7 +240,11 @@ public Work updateWork(String orcid, Work work, boolean applyAPIValidations) {
         
         if (applyAPIValidations) {
             activityValidator.validateWork(work, sourceEntity, false, applyAPIValidations, workEntity.getVisibility());
-            List<Work> existingWorks = this.findWorks(orcid, System.currentTimeMillis());            
+            
+            Date lastModified = profileEntityManager.getLastModified(orcid);
+            long lastModifiedTime = (lastModified == null) ? 0 : lastModified.getTime();
+            List<Work> existingWorks = this.findWorks(orcid, lastModifiedTime);       
```

Comment:
@amontenegro is right here - it's not something I knew.  What I did was saw that update was using findWorks and create wasn't.  So I changed them to both use it and assumed the cache would work and didn't test to check.